server.ts

```diff
@@ -166,6 +170,21 @@ const getRepoInfo = async (res: any, groupBy: string) => {
   res.status(200).json({ nodes, edges, filePaths });
 };
 
+const getRepoReadme = async (res: any) => {
+  const files = await getFilesInDirectory(REPO_PATH);
+  if(!files) {
+    res.status(404).json({ error: { message: "Repository not cloned yet" } });;
+    return;
+  }
+  const readmePth = files.find(f => f.toLowerCase().includes('readme.md'));
+  if(!readmePth) {
+    res.status(404).json({ error: { message: "Readme file not found on repository" } });;
```

Comment:
If we drop the `toLowerCase()` above we should change this to `README` instead of `Readme`